doc/toolbox.1.md

```diff
@@ -40,6 +40,20 @@ access to the user's home directory, the Wayland and X11 sockets, networking
 (including Avahi), removable devices (like USB sticks), systemd journal, SSH
 agent, D-Bus, ulimits, /dev and the udev database, etc..
 
+## Supported distribution images
+
+Toolbox maintains a list of supported distribution images. These images are
+maintained by the team to be compatible with Toolbox. Some commands accept the
+`--distro` and `--release` flags. `--distro` accepts `ID` from `os-release`
+(e.g., on Fedora it is `fedora`). `--release` accepts version number of the
+selected distribution. Each distribution uses sligthly different versioning
+schema and some don't have any at all. Supported distributions are:
+
+Name  |os-release|Format                     |URI
+------|----------|---------------------------|-----------------------------------------
+Fedora|fedora    |\<release\> or f\<release\>|registry.fedoraproject.org/fedora-toolbox
+RHEL  |rhel      |\<major\>.\<minor\>        |registry.access.redhat.com/ubi8/ubi
+
```

Comment:
The column headers should be `distro` and `release`, and we should add one or two examples.

The manuals are user-facing documentation.  Their main goal is to document the user-visible interface.  The fact that we use `os-release(5)` is secondary because it's an implementation detail.  Since this is the top-level manual, it's better to stick to the higher-level big picture items.

I wouldn't document the exact URLs of the images.  Generally speaking, anything that we document will be considered API.  It will be difficult to change those things, and we have changed URLs in the past.  Moreover, the URLs are long enough that they don't render nicely in a terminal.  Especially not in the default width of 80 columns.

Reply:
> The column headers should be `distro` and `release`, and we should add one or two examples.

Ok.
 
> The manuals are user-facing documentation. Their main goal is to document the user-visible interface. The fact that we use `os-release(5)` is secondary because it's an implementation detail. Since this is the top-level manual, it's better to stick to the higher-level big picture items.

I agree with being cautious of getting too technical but the fact we use `ID` from `os-release(5)` is very important as we require an exact value. A user needs to know what value exactly to provide.

> I wouldn't document the exact URLs of the images. Generally speaking, anything that we document will be considered API. It will be difficult to change those things, and we have changed URLs in the past. Moreover, the URLs are long enough that they don't render nicely in a terminal. Especially not in the default width of 80 columns.

Ok.

Reply:
For now went the way of leaving out the technical details.